core/model/collection.py

```diff
@@ -867,7 +870,7 @@ def delete(self, search_index=None):
         _db.commit()
 
 
-collections_libraries = Table(
+collections_libraries: Table = Table(
```

Comment:
Can you remove the `# type: ignore` on line 123, and add the same `: Table` annotation as you have here to `collections_identifiers` on line 890?

Reply:
I knew there is one more instance for this situation but couldn't find it, thanks! Added!